charts/hivemq-edge/values.schema.json

```
@@ -32,9 +32,15 @@
       "enabled" : {
         "type" : "boolean"
       },
-      "volume" : {
-        "description": "Volume containing the license.",
-        "type" : "object"
+      "volume": {
+        "licenseSecretName" : {
```

is it not a little redundant to include `license` in the name as `licenseSecretName` as the value itself is already nested in a `license` value. 
Something like
```suggestion
        "secretName" : {
```